standard/ciod_to_modules.json

```diff
@@ -5786,14 +5786,14 @@
         "moduleId":"real-time-acquisition",
         "usage":"M",
         "conditionalStatement":null,
-        "informationEntity":"Image"
```

Comment:
Is this intentional?

Reply:
No it wasn't! Adding this "Image" IE to these two objects should be a standard workaround and the unit test for it is still passing, I'll check on it.